### PR TITLE
Added CLI test for vmss with managed disks

### PR DESCRIPTION
**Pull request by soshekar**
The content to be added to Changelog is as follows:
* 
* 
*

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@soshekar__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by azurecla** [comment]
Hi __@soshekar__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: test/data/vmssParamTestMD.json [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    "networkProfile": {
+      "networkInterfaceConfigurations": [
+        {
+          "name": "test",
+          "primary": true,
+          "ipConfigurations": [
+            {
+              "name": "test",
+              "subnet": {
+                "id": "/subscriptions/24fb23e3-6ba3-41f0-9b6e-e41131d5d61e/resourceGroups/xplatTstVmssGCreate3280/providers/Microsoft.Network/VirtualNetworks/xplattestvnet6811/subnets/xplattestsubnet9872"
```
Comment (huangpf) [inline test/data/vmssParamTestMD.json]: The sub ID shall be replaced or set in the test, because every test profile has a unique sub ID.

**Comment by huangpf** [comment]
@soshekar Please sign the CLA @https://cla.azure.com/

@amarzavery @balajikris Shall we target dev branch or release branch?

**Comment by balajikris** [comment]
the `release` branch has not yet merged back into `dev`. So, either branch works. the only issue would then be merge conflicts, if we're making managed-disks related changes in both dev and release branches (for e.g, carissasun was targeting release branch) - although I don't think we have any, given the PRs I've seen so far.

**Comment by huangpf** [comment]
@balajikris But if the change was merged to release branch, it would reset the release activity, isn't it? 

So, if it's only test related change, can we keep it in dev branch?

**Comment by amarzavery** [comment]
It wouldn't reset the release activities. However, keeping the tests in dev branch is fine.

**Comment by azurecla** [comment]
__@soshekar__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

**Comment by huangpf** [comment]
@amarzavery @balajikris Tests passed. Please help merge the PR. Thanks.

**Pull request merged**